src/main/java/foxtrot/uniform/charlie/kilo/Main.java

```diff
@@ -55,19 +62,29 @@ public static void main(String[] args) {
         movieSession2.setShowTime(LocalDateTime.of(2020, 11, 30, 23, 59));
         movieSessionService.add(movieSession1);
         movieSessionService.add(movieSession2);
+        System.out.println();
+        System.out.println("\033[33mMovie Sessions\033[0m");
         System.out.println(movieSession1);
         System.out.println(movieSession2);
-        System.out.println();
-        System.out.println("\033[33mHELLO\033[0m");
-        System.out.println("\033[33mHELLO\033[0m");
-        System.out.println();
         List<MovieSession> availableSessions = movieSessionService.findAvailableSessions(
                 movie1.getId(), LocalDate.now());
         System.out.println();
-        System.out.println();
+        System.out.println("\033[33mAvailable Sessions\033[0m");
         availableSessions.forEach(System.out::println);
-        authenticationService.register("alice@gmail.com", "12345");
-        authenticationService.register("bob@gmail.com", "11111");
+        User user2 = authenticationService.register("bob@gmail.com", "11111");
+        System.out.println();
+        System.out.println("\033[33mLogging in\033[0m");
+        User user1 = authenticationService.register("alice@gmail.com", "12345");
         System.out.println(authenticationService.login("alice@gmail.com", "12345"));
+        shoppingCartService.registerNewShoppingCart(user1);
```

Comment:
this should be done automatically in `register()` method - remove it from here

Reply:
I fixed it... And I think I've done it wrong...